.craft.yml

```
@@ -1,8 +1,12 @@
 minVersion: 0.23.1
 changelogPolicy: auto
-artifactProvider:
-  name: none
 targets:
+  - name: maven
+    gradleCliPath: ./gradlew
```

```suggestion
    gradleCliPath: ./plugin-build/gradlew
```

since it has to be executed within the plugin-build folder